lib/generator.js

```diff
@@ -1887,6 +1891,14 @@
                     return this.updateCustomOption(pluginId, "websocketServerPort", actualPort)
                         .thenResolve(actualPort);
                 }.bind(this))
+                .then(function (actualPort) {
```

Comment:
This would be prettier with `tap()`, but that is not supported in this version of Q.  I'm willing to bump to v1.1 if you are... or if you have another suggestion... or just leave it.